lib/rules/indent.js

```diff
@@ -119,7 +119,10 @@ module.exports = function(context) {
     function checkNodesIndent(nodes, indent, excludeCommas) {
         nodes.forEach(function(node) {
             var nodeIndent = getNodeIndent(node, false, excludeCommas);
-            if (nodeIndent !== indent && isNodeFirstInLine(node)) {
+            if (
```

Comment:
Is there a way to make this additional check outside this function as this function is generic and is called from different portions of the code.